board.go

```diff
@@ -19,21 +19,25 @@ package trello
 import (
 	"encoding/json"
 	"fmt"
-	"io/ioutil"
-	"net/http"
+)
+
+var (
+	_ = fmt.Printf
```

Comment:
Those are dirty hacks which let me keep "fmt" in the import even if it is unused otherwise. Great for debugging where you add and remove fmt. lines so the compiler doesn't complain it is unused.  I can pull them out.